packages/p2p/src/components/i18next/index.js

```diff
@@ -1,8 +1,8 @@
-/* eslint-disable */
+/* eslint-disable import/no-unresolved */
```

Comment:
we only need to disable `import/no-unresolved` rule here. Since we importing json file with path alias. I can include json extension in `webpack.config` if its needed 🙏